src/ec/eddsa.rs

```diff
@@ -109,6 +109,11 @@ impl<'a> Ed25519KeyPair {
         Ok(pair)
     }
 
+    // Returns a reference to the little-endian-encoded private key bytes.
+    fn private_key_bytes(&'a self) -> &'a [u8] {
+        &self.private_public[..SEED_LEN]
```

Comment:
When working on the ECDSA signing stuff, I realized that eventually we should switch to `ec::PrivateKey` for holding the private scalar part of the key pair.

This code has the `private_public` member in the first place only because the original C code required the concatenated seed||public_key as an input. That's no longer the case after this PR.

So, keep in mind that in the future `private_public` will get split into multiple members, once of which is an `ec::PrivateKey`.